includes/acl/decompression/impl/decompress.impl.h

```diff
@@ -68,13 +68,14 @@ namespace acl
 	{
 		constexpr bool skip_safety_checks = decompression_settings_type::skip_initialize_safety_checks();
 
-		const bool is_valid = skip_safety_checks || tracks.is_valid(false).empty();
-		ACL_ASSERT(is_valid, "Invalid compressed tracks instance");
+		const error_result error = skip_safety_checks ? error_result() : tracks.is_valid(false);
+		const bool is_valid = skip_safety_checks || error.empty();
+		ACL_ASSERT(is_valid, "Invalid compressed tracks instance: %s", error.c_str());
```

Comment:
Minor tweak: we can't rely on '%s' as a format specifier because hosts can override the assert macro and might expect a unicode string when %s is present. To support this, we have to use `ACL_ASSERT_STRING_FORMAT_SPECIFIER` instead

Reply:
Thanks @nfrechette! This should be fixed now - I assume there isn't anything similar for `%u` I need to use?

Reply:
Yes, that's correct. Thank you!